### PR TITLE
Annotations table improvements

### PR DESCRIPTION
**Pull request by 777arc**
### What does this PR do?

See below.   Also the arrow button now works, it makes the spectrogram jump to the place in time where the annotation starts

image

### CODE REVIEW

File: client/src/Utils/rfFunctions.ts [hunk 1]
```diff
@@ -1,20 +1,40 @@
 export function calculateDate(start: Date, count: number, sampleRate: number) {
-  let time = start.getTime() + count / sampleRate;
+  let time = start.getTime() + (count / sampleRate) * 1e3; // getTime() returns milliseconds
```
Comment (777arc) [inline client/src/Utils/rfFunctions.ts:2]: @Jaydee123 btw this was the only real bug fix, turns out getTime() and Date() operate in ms not s, who knew!

File: client/src/Components/Spectrogram/SpectrogramPage.jsx [hunk 2]
```diff
@@ -802,7 +807,18 @@ class SpectrogramPage extends Component {
 
         <div className="mt-3 mb-0 ml-0 mr-0 p-0" style={{ margin: '5px' }}>
           <Collapsible title="Annotations" style={{ marginTop: '5px', width: '100%' }}>
-            <Table columns={columns} data={this.calculateData(this.state.meta)} />
+            <Table
+              columns={[
+                { title: 'Annotation', dataIndex: 'annotation' },
```
Comment (Jaydee123) [inline client/src/Components/Spectrogram/SpectrogramPage.jsx:812]: I'm not sure from a readability point of view I'm a fan of having the json for the columns here

Comment (777arc) [reply client/src/Components/Spectrogram/SpectrogramPage.jsx:812]: yeah i mean it just cant be at the very bottom of the file

Comment (Jaydee123) [reply client/src/Components/Spectrogram/SpectrogramPage.jsx:812]: Thats fine, I would usually put them at the top of the file, just forgot to move it 😁

**Review by Nepomuceno — APPROVED** [review]
Great changes 
:shipit:

**Pull request merged**